.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,19 @@
-<!--- Provide a summary of your changes in the Pull Request Title above. -->
-<!--- If this is a work in progress (not yet ready to be merged), make this a draft pull request. -->
+<!-- Provide a summary of your changes in the Pull Request Title above. -->
+<!-- If this is a work in progress (not yet ready to be merged), make this a draft pull request. -->
 
 ## Motivation and Context
 
-<!--- Why is this pull request required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here: -->
-<!--- https://docs.github.com/en/github/managing-your-work-on-github/managing-your-work-with-issues-and-pull-requests/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword -->
+<!-- Why is this pull request required? What problem does it solve? -->
+<!-- If it fixes an open issue, link the issue by typing in 'resolves #' then the issue number like (resolves #709) -->
```

Comment:
I deliberately linked the documentation so that you would read it. Revert this change.